easybuild/tools/package/utilities.py

```diff
@@ -110,6 +111,18 @@ def package_with_fpm(easyblock):
         dep_pkgname = package_naming_scheme.name(dep)
         depstring += " --depends %s" % quote_str(dep_pkgname)
 
+    # Excluding the EasyBuild logs and test reports that might be in the installdir
+    exclude_files_glob = [
+        os.path.join(log_path(), "*.log"),
+        os.path.join(log_path(), "*.md"),
+    ]
+    # stripping off leading / to match expected glob in fpm
+    exclude_files_glob = [
+        '--exclude %s' % quote_str(os.path.join(easyblock.installdir[1:], x))
```

Comment:
use `easyblock.installdir.lstrip(os.pathsep)`

you have no idea what you're stripping off with `[1:]`